src/main/java/de/komoot/photon/utils/ConvertToJson.java

```diff
@@ -76,6 +77,13 @@ private String getLocalised(Map<String, Object> source, String fieldName, String
             return map.get(lang);
         }
 
+        if (fieldName.equals("name")) {
+            for (String key : NAME_PRECEDENCE) {
+                if (map.containsKey(key))
+                    return map.get(key);
+            }
+        }
```

Comment:
But now 'default' is the falllback, while it was the first choice before.

Reply:
If `default` needs to be the first preference, I see two ways of doing it:

1. Add default as the first entry in the NAME_PRECEDENCE array.
2. Change the `if` condition from `(fieldName.equals("name"))` to `(map.get("default") == null && fieldName.equals("name"))`.

Which option should I go for?